third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2159,8 +2159,68 @@ def map(self, func, na_action: Optional[str] = None) -> DataFrame:
            In pandas 2.1.0, DataFrame.applymap is deprecated and renamed to
            DataFrame.map.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            Let's use ``reuse=False`` flag to make sure a new ``remote_function``
```

Comment:
Only the code sample itself should be indented 4 spaces. That's so we can trick Cloud RAD into using a preformatted text font. This text doesn't / shouldn't be in that code-style font.

Reply:
Thanks for the catch, fixed!